feat(javascript): Link to Github `MIGRATION.md` file from docs (#5005)
getsentry/sentry-docs: src/includes/getting-started-install/javascript.mdx

We also support alternate [installation methods](/platforms/javascript/install/).

</Note>

<Note>

> Are you updating your Sentry SDK to the latest version? Check out our [migration guide](https://github.com/getsentry/sentry-javascript/blob/7.x/MIGRATION.md) to learn more about breaking changes!

Reviewer: I like this!

> <Note>
> 
> Are you updating your Sentry SDK to the latest version? Check out our [migration guide](https://github.com/getsentry/sentry-javascript/blob/7.x/MIGRATION.md) to learn more about breaking changes!
> 
> </Note>

Reviewer 2: Suggest changing the language here so we can take this out of a note and make it body text. Right now, it comes right after another note that's already on the page, and having two notes in a row with no body text between them is something we should avoid. Also, we don't use exclamation points in docs.

```suggestion
 If you're updating your Sentry SDK to the latest version, check out our [migration guide](https://github.com/getsentry/sentry-javascript/blob/7.x/MIGRATION.md) to learn more about breaking changes.

```
